lib/mutant/mutator/node/regexp/greedy_zero_or_more.rb

```diff
@@ -0,0 +1,24 @@
+module Mutant
+  class Mutator
+    class Node
+      module Regexp
+        # Mutator for greedy zero-or-more quantifier, `*`
+        class GreedyZeroOrMore < Node
+          handle(:regexp_greedy_zero_or_more)
+
+          children :min, :max, :subject
+
+          # Emit mutations
+          #
+          # Replace `/a*/` with `/a+/`
+          #
+          # @return [undefined]
+          def dispatch
+            emit(s(:regexp_greedy_one_or_more, *children))
+            emit_subject_mutations
```

Comment:
There is an explicit repetition syntax: `a{1,2}` so I assume that this attributes are leaked from that node.